styles/base.css

```diff
@@ -24,7 +24,7 @@ body {
     display: flex;
     flex-direction: column;
 
-    padding: 30px;
+    padding: 16px;
```

Comment:
I'm curious about this change? Personally I like a little more breathing room around the edges.

Reply:
This change was to match the spacing specified in Figma for mobile. I agree though that 16px looks a bit too close to the edge for desktop.

In [f0d43fa](https://github.com/compilerla/compiler.la/pull/32/commits/f0d43fa88ab4a589eb9b837da2da2025bf660e97), I've put back the 30px padding for desktop and added to the existing media-query so mobile has 16px padding.

Reply:
I'll update in Figma!